docs/pages/admin-guides/management/operations/backup-restore.mdx

```diff
@@ -2,211 +2,118 @@
 
-# bootstrap fresh server (kill the old one first!)
-$ sudo teleport start --config fresh.yaml --bootstrap state.yaml
+### Using a state file
 
-# from another terminal, verify state transferred correctly
-$ tctl --config fresh.yaml get all
-# <your state here>
-```
+You can run a command to retrieve all resource manifests and print them to a
+single file, the **state file**, then restart the Teleport Auth Service using
+the state file to bootstrap the cluster.
 
-The `--bootstrap` flag has no effect, except when the Auth Service initializes
-its backend initialization on first startup, so it is safe for use in
-supervised/High Availability contexts.
-
-### Limitations
-
-The `--bootstrap` flag doesn't re-trigger Trusted Cluster handshakes, so Trusted
-Cluster resources need to be recreated manually.
-
-All the same limitations around modifying the config file of an existing cluster
-also apply to a new cluster being bootstrapped from the state of an old cluster:
-
-  - Changing the cluster name will break your CAs. This will be caught and Teleport
-    will refuse to start.
-  - Some user authentication mechanisms (e.g. WebAuthn) require that the public
-    endpoint of the Web UI remains the same. This cannot be caught by Teleport,
-    so be careful!
-  - Any Node whose invite token is defined in the Auth Service's configuration
-    file will be able to join automatically, but Nodes that were added
-    dynamically will need to be re-invited.
+#### Restarting the Auth Service with a state file
 
-</TabItem>
-<TabItem scope={["oss"]} label="Open Source">
+1. Log in to your Auth Service machine and run the following command:
 
-You can export a collection of resources from
-Teleport using the below command. This feature helps for migrating
-from one backend to another.
+   ```code
+   $ tctl get all --with-secrets > state.yaml
+   ```
 
-Using `tctl get all --with-secrets` will retrieve the below items:
+1. Prepare a new uninitialized backend (make sure to port any non-default config
+   values from the old config file):
 
-- Users
-- Certificate Authorities
-- Trusted Clusters
-- GitHub Connectors
-- Roles
+   ```code
+   $ mkdir fresh && cat > fresh.yaml << EOF
+   teleport:
+     data_dir: fresh
+   EOF
+   ```
 
-When migrating backends, you should back up your Auth Service's
-`data_dir/storage` directly.
+1. Stop Teleport on each Auth Service instance.
 
-### Example of backing up and restoring a cluster
+1. Restart Teleport on each Auth Service instance with the fresh backend and the
+   state file you created earlier:
 
-```code
-# Log in to your cluster with tsh so you can use tctl from your local machine.
-# You can also run tctl on your Auth Service host without running "tsh login"
-# first.
-$ tsh login --user=myuser --proxy=teleport.example.com
-# Export dynamic configuration state from old cluster
-$ tctl get all --with-secrets > state.yaml
+   ```code
+   $ sudo teleport start --config fresh.yaml --bootstrap state.yaml
```

Comment:
This might confuse people who use one of our systemd-based package installs.

Are we suggesting that users
- stop the systemd service
- start teleport manually with `--bootstrap` (not using systemd)
- stop the manually started version and restart the systemd service
?

Reply:
👍 I've removed the `--bootstrap` discussion and added a section about `teleport backend clone`.